src/main/java/at/hannibal2/skyhanni/config/features/skillprogress/SkillProgressBarConfig.java

```diff
@@ -0,0 +1,107 @@
+package at.hannibal2.skyhanni.config.features.skillprogress;
+
+import at.hannibal2.skyhanni.SkyHanniMod;
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.Accordion;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorColour;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorDropdown;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorSlider;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+import io.github.moulberry.moulconfig.observer.Property;
+
+public class SkillProgressBarConfig {
+
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Enable/Disable the progress bar.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public Property<Boolean> enabled = Property.of(false);
+
+    @Expose
+    @ConfigOption(name = "Textured Bar", desc = "Use a textured progress bar.\n§eCan be changed with a resource pack.")
+    @ConfigEditorBoolean
+    public Property<Boolean> useTexturedBar = Property.of(false);
+
+    @Expose
+    @ConfigOption(name = "Chroma", desc = "Use the SBA like chroma effect on the bar.\n§eIf enabled, ignore the Bar Color setting.")
+    @ConfigEditorBoolean
+    public Property<Boolean> useChroma = Property.of(false);
```

Comment:
The contrast in the progress bar is very weak by default. Why don't we leave the not filled part in a way darker color instead of just a bit darker/brighter? If there is no good way to combine color picker with high contrast, maybe make chroma default enabled, or even remove the custom color thing entirely?
As comparison, see the chroma effect. Way better contrast.
![grafik](https://github.com/hannibal002/SkyHanni/assets/24389977/b645b897-06a7-4ec9-b140-58e737548470)
![grafik](https://github.com/hannibal002/SkyHanni/assets/24389977/7c9a3b5e-95ec-4013-af3d-2b12fb4cd914)